packages/groqd-playground/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "groqd-playground",
```

Comment:
Thoughts on name before we ship the first release?

Reply:
Either `playground` or `explorer`